src/registry/create.ts

```diff
@@ -25,6 +25,7 @@ export async function prepareCreateTable({
     `${first}_${chainId}`,
     true
   );
+  statement = statement.trim();
```

Comment:
```suggestion
  // Trim whitespace to avoid issues with regex later
  statement = statement.trim();
```

Reply:
Removed this trim in favor of a trim in the Statement constructor.